src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -638,6 +638,7 @@
             color: $gray-dark;
             // FIXME: this should be in the general reboot
             cursor: pointer;
+            outline: none;
```

Comment:
I don't think we can use `outline: none` because then the pagination links become inaccessible via keyboard.